editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -75,6 +75,26 @@
 #define MIN_FOV 0.01
 #define MAX_FOV 179
 
+namespace {
```

Comment:
We don't normally use namespaces in Godot code. This should likely just be a normal static method or a private class method.

Reply:
Ok. Replaced namespace with a static function.